modules/catalog/src/test/java/org/apache/ignite/internal/catalog/commands/AbstractCommandValidationTest.java

```diff
@@ -193,13 +194,14 @@ static CatalogCommand createZoneCommand(String zoneName, List<String> storagePro
     }
 
     static Catalog applyCommandsToCatalog(Catalog catalog, CatalogCommand... commandsToApply) {
+        UpdateContext updateContext = new UpdateContext(catalog);
         for (CatalogCommand command : commandsToApply) {
-            for (UpdateEntry updates : command.get(catalog)) {
-                catalog = updates.applyUpdate(catalog, INITIAL_CAUSALITY_TOKEN);
+            for (UpdateEntry updates : command.get(updateContext)) {
+                updateContext = new UpdateContext(updates.applyUpdate(updateContext.catalog(), INITIAL_CAUSALITY_TOKEN));
```

Comment:
Why don't reuse updateContest instance and call `UpdateContext.updateCatalog()` instead.

Reply:
Done, thanks